src/twisted/newsfragments/9972.feature

```diff
@@ -0,0 +1 @@
+add twisted.internet.defer.coroFnToDeferredFn decorator for convenient use of async def functions
```

Comment:
I think it is customary to make this a full sentence, so starting with a capital and ending with a period.

Reply:
```suggestion
Add twisted.internet.defer.coroFnToDeferredFn decorator for convenient use of async def functions.
```